tools/topology/topology2/cavs/cavs-rt5682-rt1019.conf

```diff
@@ -0,0 +1,250 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-playback.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+
+#
+# List of all DAIs
+#
+# PCM0 ----> volume -----> SSP1  (Speaker - ALC1019)
+# PCM1 <---> volume <----> SSP0  (Headset - ALC5682)
+# PCM2 <---- volume <---- DMIC01 (dmic 48k capture)
+Object.Dai {
+	SSP."0" {
+		# sof_rt5682.c in machine driver - id for speaker is 6 - 1 headset + 2 dmic + 3 hdmi (if not defined in quirks)
+		id 		6
+		direction	"duplex"
+		name		SSP0-Codec
+		default_hw_conf_id	0
+		sample_bits		24
+
+		Object.Base.hw_config."SSP0" {
+			id	0
+			mclk_freq	24000000
+			bclk_freq	4800000
+			tdm_slot_width	25
+		}
+
+		# include DAI copier components
+		Object.Widget.copier."0" {
+			index 1
+			dai_index 1
+			type "dai_in"
+			dai_type "SSP"
+			copier_type "SSP"
+			direction "playback"
+			stream_name "SSP0-Codec"
+			period_sink_count 0
+			period_source_count 2
+			format s24le
+		}
+
+		Object.Widget.copier."1" {
+			index 2
+			dai_index 2
+			type "dai_out"
+			dai_type "SSP"
+			copier_type "SSP"
+			direction "capture"
+			stream_name "SSP0-Codec"
+			period_sink_count 2
+			period_source_count 0
+			format s24le
+		}
+	}
+	SSP."1" {
+		id 		0
+		direction	"playback"
+		name		SSP1-Codec
+		default_hw_conf_id	0
+		sample_bits		24
+
+		Object.Base.hw_config."SSP1" {
+			id	0
+			mclk_freq	24000000
+			bclk_freq	3072000
```

Comment:
that means mclk and bclk use a different source. I thought there were some changes added in TGL and follow-ups, does this work?

Reply:
??

Reply:
Updating all settings similar to rt1011

https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-tgl-max98357a-rt5682.m4#L380-L385